src/main/java/com/exasol/adapter/document/UdfCallBuilder.java

```diff
@@ -83,31 +95,31 @@ private Select wrapStatementInStatementWithPostSelectionAndProjection(final List
      * statement.
      */
     private Select buildUdfCallStatement(final List<DocumentFetcher<DocumentVisitorType>> documentFetchers,
-            final RemoteTableQuery query, final String connectionName) throws IOException {
+            final RemoteTableQuery query) throws IOException {
         final Select doubleNestedSelect = StatementFactory.getInstance().select();
         final List<Column> emitsColumns = query
                 .getRequiredColumns().stream().map(column -> new Column(doubleNestedSelect,
                         column.getExasolColumnName(), convertDataType(column.getExasolDataType())))
                 .collect(Collectors.toList());
-        doubleNestedSelect.udf("Adapter." + ImportDocumentData.UDF_NAME, new ColumnsDefinition(emitsColumns),
-                column(DOCUMENT_FETCHER_PARAMETER), column(REMOTE_TABLE_QUERY_PARAMETER),
-                column(CONNECTION_NAME_PARAMETER));
-        final ValueTable valueTable = buildValueTable(documentFetchers, query, connectionName, doubleNestedSelect);
+        doubleNestedSelect.udf("Adapter." + ImportDocumentData.UDF_PREFIX + this.adapterName,
+                new ColumnsDefinition(emitsColumns), column(DOCUMENT_FETCHER_PARAMETER),
+                column(REMOTE_TABLE_QUERY_PARAMETER), column(CONNECTION_NAME_PARAMETER));
```

Comment:
This part is hard to read. Please extract method(s) with speaking name(s).

Reply:
better like this?